sections/introduction.tex

```diff
@@ -0,0 +1,64 @@
+\section{Introduction}\label{sec:introduction}
+% Intro to recommendation and techniques
+News is being broadcast every day around the world in the form of news articles, television, and newspapers, which supply people with the latest information.
+Searching and categorizing the news is becoming a bigger problem since news is created at all times.
+As the internet becomes more personalized, as seen with targeted advertisements, recommender systems are seeing more use in applications, such as Netflix and Amazon.
+Within recommendation, there are two main machine learning techniques: collaborative filtering and content-based filtering.
+
+One particular field, which covers a variety of different use cases and applications, such as being useful for recommendations, is the field of topic modeling. \vejleder{gerne mere skarp i forhold til anbefalinger}
+Topic modeling has been used for finding topics within a collection of documents to either annotate or categorize certain collections of texts~\cite{Probabilistic_Topic_Models}.
+The \gls{lda} model is a well-cited topic model which generates topics and topic distributions for documents based on the words in documents~\cite{blei2003latent}.
+Extensions of \gls{lda} have also been proposed to model various other information sources to generate better topics and/or topics, however, with different focuses and potential uses.
+In most models metadata is not used.\vejleder{er dette korrekt? + noget mere jeg ikke kan læse}
+Author-Topic model by \citet{author_topic_2012} and the MetaLDA model by \citet{MetaLDA2017} being notable examples.
+
+% Author-Topic
+The Author-Topic model by \citet{author_topic_2012} combines the \gls{lda} model with author information to model the relationship between authors and the documents, which they have written.
+This is based on the assumption that most authors usually write about only a few different topics.
+By modeling this relationship, connections between authors and topics, and between different authors, can be inferred giving more underlying information to the topics.
```

Comment:
Føler beskrivelsen i intro bliver lidt lang. Måske fjern denne sætning, og ændre til kun at nævne den ene observation. "They show that the ..."

Reply:
```suggestion
They show that by modeling this relationship, connections between authors and topics, and between different authors, can be inferred giving more underlying information to the topics. 
```